docs/USAGE-GUIDE.md

```diff
@@ -1560,6 +1566,46 @@ handshake.
 
 **s2n_connection_is_session_resumed** returns 1 if the handshake was abbreviated, otherwise returns 0, for tls versions < TLS1.3.
 
+## Additional TLS1.3 Session Resumption Related calls
+
+Session resumption works differently in versions TLS1.3 and higher. Session ticket messages are now sent after the
+handshake in "post-handshake" messages. While some of the TLS1.2 session resumption APIs are still relevant for TLS1.3
```

Comment:
```suggestion
Session resumption works differently in versions TLS1.3 and higher. Session ticket messages are now sent after the
handshake in "post-handshake" messages, and multiple session tickets may be issued for the same connection. While some of the TLS1.2 session resumption APIs are still relevant for TLS1.3
```

Reply:
is there a good way to tell which apis are still relevant for tls1.3 and which are not? is it documented somewhere?